src/models/differentiable/evotree_model.jl

```diff
@@ -0,0 +1,147 @@
+using DataFrames
+using SliceMap
+using EvoTrees
+using MLJBase
+
+"""
+This type provides a basic interface to gradient-boosted tree models from the MLJ library.
+However, this is not be the final version of the interface: full support for EvoTrees has not been implemented yet and the `logits` and `probs` methods will be changed in the process of doing that if needed.
+"""
+
+"""
+    EvoTreeModel <: AbstractMLJModel
+
+Constructor for gradient-boosted decision trees from the EvoTrees.jl library. 
+"""
+struct EvoTreeModel <: AbstractDifferentiableModel
```

Comment:
And here you're not even using this `AbstractMLJModel` struct, also the comment is wrong

Reply:
Right, good catch. I'll change this based on our final decision on whether to retain the AbstractMLJModel class or not.